src/docs/cmdstan-guide/diagnose.tex

```diff
@@ -0,0 +1,112 @@
+\chapter{{\tt\bfseries diagnose}: Output Sanity Checking}\label{diagnose.chapter}
+
+\noindent
+\CmdStan is distributed with a utility that is able to read in and
+analyze the output of one or more Markov chains to check for the
+following potential problems:
+
+\begin{itemize}
+\item Transitions that hit the maximum treedepth
+\item Divergent transitions
+\item Low E-BFMI values
+\item Low effective sample sizes
+\item High $\hat{R}$ values
+\end{itemize}
+
+The meanings of several of these problems are discussed in
+\url{http://mc-stan.org/misc/warnings.html#runtime-warnings}.
```

Comment:
Additionally please include a reference to the review paper, https://arxiv.org/abs/1701.02434.